src/elm/Styles/Colours.elm

```diff
@@ -0,0 +1,78 @@
+module Styles.Colours exposing (getPastel)
+
+import Styles.Classes exposing (tachs)
+import Color exposing (..)
+import Array exposing (..)
+import Maybe exposing (withDefault)
+
+
+type alias Swatches =
+    Array RgbString
+
```

Comment:
Yeah the Array type is set up to simply index the swatches, to use the 'get' method.List doesn't have indexing by default (though the *List.Extra* library looks great, and will likely be imported for future view functions...).

I definitely am keen for using Sets in constructors to make more organised libraries in Colours.elm. To account assert uniqueness without sets, I've refactored the *buildSwatches* function by spreading the *total* input across the hue degree range when generating *hsl* colours in my latest commit